pinecone/client.go

```diff
@@ -280,6 +280,10 @@ func (c *Client) Index(in NewIndexConnParams, dialOpts ...grpc.DialOption) (*Ind
 		in.AdditionalMetadata = make(map[string]string)
 	}
 
+	if in.Host == "" {
+		return nil, fmt.Errorf("host is required to create an IndexConnection. Find your host from calling DescribeIndex or via the Pinecone console")
```

Comment:
Can we capitalize `Host` here since that's how the variable is defined on the struct. We could also call out `ListIndexes` since you can see those values through that as well.